oss-cloud-infrastructure/serverless.yml

```diff
@@ -0,0 +1,11 @@
+service: tst-infrastructure
```

Comment:
If "tst" stands for test lets rename it. For example: "oss-infrastructure" ? The same applies to SSM parameters above.